packages/plugin-ext/src/main/browser/webview/webview-context-keys.ts

```diff
@@ -0,0 +1,49 @@
+// *****************************************************************************
+// Copyright (C) 2023 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { inject, injectable, postConstruct } from '@theia/core/shared/inversify';
+import { ContextKey, ContextKeyService } from '@theia/core/lib/browser/context-key-service';
+import { ApplicationShell, FocusTracker, Widget } from '@theia/core/lib/browser';
+import { WebviewWidget } from './webview';
+
+@injectable()
+export class WebviewContextKeys {
+
+    /**
+     * Context key representing the `viewType` of the active `WebviewWidget`, if any.
+     */
+    activeWebviewPanelId: ContextKey<string | undefined>;
```

Comment:
It looks like the VSCode version is always a string, and the default value is an empty string.